package.json

```diff
@@ -1,7 +1,7 @@
 {
     "type": "commonjs",
     "name": "cyclic-dynamodb",
-    "version": "0.0.20",
+    "version": "0.0.22",
```

Comment:
Something is out of sync here `.22` but package lock is `.21`

Reply:
already in npm